weblate_web/templates/download.html

```diff
@@ -14,7 +14,7 @@
         <div class="wrap">
             <h1 class="section-title min-m">{% trans "Download Weblate" %}</h1>
             <div class="page-desc">
-                <p>{% trans "Weblate comes with complete source code that you can use, edit or alter to meet your needs. Feel free to let your associates know about Weblate." %}</p>
+                <p>{% trans "All Weblate source code is freely available for you to use, modify, share and distribute. Discuss it with your associates." %}</p>
```

Comment:
Possibly "Let your associates know"?

Reply:
Hard to tell for me, this goes beyond by English knowledge...